keras_nlp/models/bert/bert_presets.py

```diff
@@ -34,6 +34,14 @@
             "Tiny size of BERT where all input is lowercased. "
             "Trained on English Wikipedia + BooksCorpus."
         ),
+        "metadata":{
+            "description": (
+                "Tiny size of BERT where all input is lowercased. "
+                "Trained on English Wikipedia + BooksCorpus."
+            ),
+            "params": "4M",
+            "models": "[BERT](bert)"
```

Comment:
We probably don't need this in the metadata, we can add this part directly from the rendering code on keras.io